RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -81,18 +84,36 @@ def TICL_iterations_withReco(process):
       LayerClustersInputMask = "trackstersMIP"
   )
 
-  process.tracksters = trackstersProducer.clone(
+  process.trackstersEM = trackstersProducer.clone(
       original_mask = "trackstersMIP",
       filtered_mask = cms.InputTag("filteredLayerClusters", "algo8"),
       seeding_regions = "ticlSeedingGlobal",
       missing_layers = 2,
-      min_clusters_per_ntuplet = 15,
+      min_clusters_per_ntuplet = 10,
       min_cos_theta = 0.94, # ~20 degrees
-      min_cos_pointing = 0.7
+      min_cos_pointing = 0.7,
+      out_in_dfs = True,
+      max_out_in_hops = 10,
```

Comment:
these are identical to what's already in TrackstersProducer, no need to "modify"